Documentation/operations/system_requirements.rst

```diff
@@ -98,6 +98,16 @@ RancherOS_                 >= 1.5.5
           Linux distribution that works well, please let us know by opening a
           GitHub issue or by creating a pull request that updates this guide.
 
+.. note:: Systemd 245 and above (``systemctl --version``) overrides ``rp_filter`` setting
+          of Cilium network interfaces. This introduces connectivity issues (see
+          `GH-10645 <https://github.com/cilium/cilium/issues/10645>`_ for details). To
+          avoid that, the following can be run:
+
+          .. code:: bash
+
+              echo 'net.ipv4.conf.lxc*.rp_filter = 0' >> /etc/sysctl.d/90-override.conf
```

Comment:
I think it's nicer to put these in a separate file really, and shouldn't `cilium_*` interface have this setting also?

Here's what I did earlier and it worked well: https://github.com/cilium/cilium/issues/10645#issuecomment-707215056

Reply:
Interesting, in my case it works fine when only lxc* is blacklisted. What is your configuration? Have you tried w/o `cilium_*`?

Reply:
No, I've not tried, but my understanding was that Cilium's intention was to disable `rp_filter` everywhere.

Reply:
When I search for references to `rp_filter`, I find multiple things: https://github.com/cilium/cilium/search?q=rp_filter

I am actually not so sure what to make of it. I guess its possible that it matters to `lxc_*` interfaces.